tests/test-program.py

```diff
@@ -194,38 +200,47 @@ def transform_messages(self, testcase, context=None):
 
     def test_basic(self):
         """ test basic message """
-        self.transform_messages('basic')
-        self.transform_messages_streaming('basic')
+        self.transform_messages({ 'testcase': 'basic'})
+        self.transform_messages_streaming({ 'testcase': 'basic'})
+
+    def test_basic_no_config(self):
+        """ test basic no config message """
+        schemas = {
+            'input': 'schemas/examples-messages.output.json',
```

Comment:
Is this correct? To use `examples-messages.output.json` instead of `examples-messages.input.json`

Reply:
Yes, in that it's how all of the other tests are setup - the input/output schema winds up being functionally the same.

Reply:
That said.....

Reply:
Let's 'fix' that: 7620bac